170327-nipype/notebooks/rfmri_group_1/PreprocessingWorkflow.ipynb

```diff
@@ -2,21 +2,102 @@
  "cells": [
   {
    "cell_type": "code",
-   "execution_count": 2,
-   "metadata": {},
+   "execution_count": 5,
```

Comment:
Please git checkout cclemens/group1 -- 170327-nipype/notebooks/rfmri_group_1/PreprocessingWorkflow.ipynb to rewrite this file